mm2src/coins/lightning.rs

```diff
@@ -754,6 +754,11 @@ impl SwapOps for LightningCoin {
         utxo_common::derive_htlc_key_pair(self.platform.coin.as_ref(), swap_unique_data)
     }
 
+    #[inline]
+    fn derive_htlc_pubkey(&self, swap_unique_data: &[u8]) -> Vec<u8> {
+        utxo_common::derive_htlc_pubkey(self, swap_unique_data)
+    }
+
```

Comment:
I'm unsure if it's correct to use `utxo_common` implementation here. Will wait for @shamardy comment 🙂

Reply:
The htlc pubkeys or key pairs returned from this function or `derive_htlc_key_pair` https://github.com/KomodoPlatform/atomicDEX-API/blob/31f427554a2a616fc264db8862ef54a3d05b1a4d/mm2src/coins/lightning.rs#L753-L755 are not used in lightning swaps since payment instructions (Invoices) are used instead, that's why I used platform coin key pair as a placeholder. But I see that this leaks info about the platform coin pubkey, so it would be better to use the lightning node pubkey here instead.
We can leave this as is in this PR, and I will fix it in another PR. I added this comment to the lightning integration checklist https://github.com/KomodoPlatform/atomicDEX-API/issues/1045#issue-971642359.

Reply:
I can easily fix this, thanks!

Reply:
> it would be better to use the lightning node pubkey here instead

Looks good after this commit https://github.com/KomodoPlatform/atomicDEX-API/pull/1591/commits/a85eb6c9451009784631a6b47863404d9313dcbb